fenix/app/src/main/java/org/mozilla/fenix/tabstray/TabsTray.kt

```diff
@@ -92,6 +94,7 @@ fun TabsTray(
     appStore: AppStore,
     browserStore: BrowserStore,
     tabsTrayStore: TabsTrayStore,
+    storage: ThumbnailStorage?,
```

Comment:
I think there's an interesting conversation we could have about prop drilling utility classes like this down the whole tree vs fetching them from within it (at an appropriate level to avoid problems like the one you're solving). This ties in pretty closely to our ongoing conversation about state hoisting as well, since this problem would've been avoided if we followed that advice to the letter.

IMO it would still be fine to fetch it from `LocalContext` at a lower level in the tree than this, since it is a stateless utility class. For simplicity and consistency I think we should keep this patch as it is now, but this could be a good example to use in future conversations about hoisting.